src/scripts/controller/ActivityController.js

```diff
@@ -39,7 +58,18 @@ export default class ActivityController {
   }
 
   onFinish() {
-    console.log('onFinish: ', this.path);
+    console.log('onFinish');
```

Comment:
Was going to drop this line in Matt's branch but looks like this is consistent with the console logging for `onUpdate`, `onStart` and so on. Yay consistency :tada: